helpers/github.ts

```diff
@@ -62,6 +62,22 @@ export async function forceCloseMissingIssues(devpoolIssues: GitHubIssue[], proj
   }
 }
 
+/**
+ * Stops forks from spamming real Ubiquity issues with links to their forks
+ * @returns true if the authenticated user is Ubiquity
+ */
+export async function isUbq() {
```

Comment:
```suggestion
export async function checkForUbiquityOrganization() {
```

Shorthand is unnecessary to enhance code readability. Besides, UBQ is the name of our governance token and should not be used to refer to the organization name.